resolwe/flow/elastic_indexes/data.py

```diff
@@ -0,0 +1,83 @@
+import elasticsearch_dsl as dsl
+
+from resolwe.elastic.builder import ManyToManyDependency
+from resolwe.elastic.indices import BaseIndex
+from resolwe.elastic.fields import ProcessType
+
+from .base import BaseDocument, BaseIndexMixin
+from ..models import Data
+
+
+class DataDocument(BaseDocument):
+    """Document for data search."""
+
+    started = dsl.Date()
+    finished = dsl.Date()
+    status = dsl.String(index='not_analyzed')
+    process = dsl.Integer()
+    process_type = ProcessType()
+    process_name = dsl.String()
+    tags = dsl.String(multi=True, index='not_analyzed')
+
+    collection = dsl.Integer(multi=True)
+    parents = dsl.Integer(multi=True)
+    children = dsl.Integer(multi=True)
+    entity = dsl.Integer(multi=True)
+
+    # Output fields (may not exist on all data objects).
+    source = dsl.String(index='not_analyzed')
```

Comment:
This does not belong into resolwe.

Reply:
Oh good point. Hm, this is annoying as we would need to augment the index in resolwe-bio?